assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js

```diff
@@ -85,6 +91,14 @@ function DashboardTopEarningPagesWidget( { Widget, WidgetReportZero, WidgetRepor
 		/>
 	);
 
+	if ( isAdblockerActive ) {
+		return (
+			<Widget>
```

Comment:
@johnPhillips please, add a footer to the `Widget` element to make it consistent with other states of this widget.

Reply:
Sure thing. I wasn't sure whether a footer with SourceLink was appropriate here...